handbook/ldap/ldap.md

```diff
@@ -0,0 +1,118 @@
+# Sign on with LDAP
+
+You can configure SystemLink to use the Lightweight Directory Access Protocol (LDAP) for user authentication. LDAP attributes and groups can be used to map users to roles and workspaces in SystemLink's role based access control system.
+
+## Assumptions and Prerequisites
+
+- A server running SystemLink. Refer to [Installing and Configuring SystemLink Server and Clients](https://www.ni.com/documentation/en/systemlink/latest/setup/configuring-systemlink-server-clients/) for the basics of setting up a SystemLink server.
+
+- Administrator desktop access to the SystemLink server.
+
+- An LDAP server accessible to the SystemLink server.
+
+- A bind user and bind password for the LDAP server.
+
+- Familiarity with the LDAP attributes available to your organization. If you do not know what LDAP attributes and groups are available to you please contact the system administrator for your LDAP server.
+
+## Enabling LDAP in SystemLink
+
+### Configuring Authentication
+
+1. Log into the server running SystemLink and open **NI Web Sever Configuration**.
+
+2. Go to the **Authentication** tab and enable **Connect to an LDAP server**.
+
+3. Enter the LDAP url for your sever. Refer to [**LDAP URLs**](#ldap-urls) for details on how to structure this URL.
+
+4. Enter the bind user. Refer to [**Bind User**](#bind-user) for details on usernames that can be used here.
+
+5. Enter the bind password.
+
+6. Click **Apply and Restart**.
+
+<figure>
+  <img src="../../img/ldap-setup.png" width="500" />
+  <figcaption>Configuration needed to enable logging into a SystemLink server using LDAP credentials.</figcaption>
+</figure>
+
+You may now login to the SystemLink server using your LDAP credentials. Please note that you will not be able to access systems and data in SystemLink unless you have configured workspace and role mappings. Refer to [**Mapping LDAP Attributes and Groups to SystemLink Workspaces and Roles**](#mapping-ldap-attributes-and-groups-to-systemlink-workspaces-and-roles) for details.
```

Comment:
>Please note that you will not be able to access systems and data in SystemLink unless you have configured workspace and role mappings. 

To access systems and data in SystemLink, configure workspace and role mappings.